### ISSUE TITLE
`unexpected-spaces-around-keyword-parameter-equals` (`E251`) - conflicts with formatter / false positive on python 3.13 type var defaults

### ISSUE THREAD

**Issue by DetachHead** [astral-sh/ruff#13699]
```py
def foo[T = int](): ... # Unexpected spaces around keyword / parameter equals
```
https://play.ruff.rs/ae847913-3125-4fe5-88a5-376630dfd337

i think it's best to keep the spaces because it would look strange especially when used with bounds:

```py
def foo[T: int | str=int](): ...
```

**Comment by KotlinIsland** [comment]
it type parameters should be consistent with value parameters
```py
def foo[T: int = 1](t: int = 1): ...
```
